docs/reserved-keywords-spec-version-2019.adoc

```diff
@@ -0,0 +1,294 @@
+= PartiQL Reserved Keywords
+
+The PartiQL language uses reserved keywords to parse and understand PartiQL statements.
+PartiQL Language Grammar Reserved Keywords cannot be used in PartiQL statements as-is.
+Example 1 shows queries that result in _Syntax Error_ because `FROM` is a Reserved Keyword in PartiQL:
+
+.Example 1. Invalid PartiQL Query with a Reserved Keyword
+[source, sql]
+----
+SELECT t.FROM FROM <<{'ID': 1, 'FROM': 'NA'}, {'ID': 2, 'FROM': 'SA'}>> AS t;
+-- Output: Syntax Error
+
+-- A reference to a Reserved Keyword is case-insensitive
+SELECT t.from FROM <<{'ID': 1, 'FROM': 'NA'}, {'ID': 2, 'FROM': 'SA'}>> AS t;
+-- Output: Syntax Error
+----
+
+In order to use Reserved Keywords in your PartiQL statements, use Quoted Identifiers as Example 2 shows:
+
+.Example 2. Using Quoted Identifier as a Reserve Keyword in a PartiQL Statement.
+[source, sql]
+----
+SELECT t."FROM" FROM <<{'ID': 1, 'FROM': 'NA'}, {'ID': 2, 'FROM': 'SA'}>> AS t;
+
+-- Output: <<{'FROM': 'NA'}, {'FROM': 'SA'}
+----
+
+_Note: In PartiQL a Quoted Identifier is case-sensitive; because of this rule, no result will get returned from the query shown in Example 3._
```

Comment:
minor: for this example, couldn't the result depend on if schema is present and if the PartiQL implementation will error when it can guarantee the result is always MISSING (see [section 4.1.1 of the PartiQL spec](https://partiql.org/assets/PartiQL-Specification.pdf#subsubsection.4.1.1))?

Reply:
The assumption there is that there is not schema in the environment but I can make that clear.